specification/purview/resource-manager/Microsoft.Purview/preview/2023-05-01-preview/purview.json

```diff
@@ -0,0 +1,2980 @@
+{
```

Comment:
What are the new changes in this API Version from previous stable/preview API versions?

Please create a base commit first, which has same changes as of the previous stable/preview API version and make new changes in next set of commits. That helps ARM Reviewers review the differences between last and first commits.

Reply:
Following changes are new in this preview API version from the previous stable API version -
1. Add new property: `ingestionStorage` in public Account model.
2. Add new endpoint: GET `ingestionPrivateEndpointConnections`.
3. Add new endpoint: POST `ingestionPrivateEndpointConnectionStatus`.
4. Add enum value: `S3` in `scopeType` enum of Consent properties.
5. Removal of property: `guardian` property in public Account.endpoints model, as it is not supported or used anymore.

I have updated the commits to have a base commit of existing API versions.